packages/neutron-sdk/src/interchain_queries/v045/queries.rs

```diff
@@ -135,6 +154,27 @@ pub fn query_staking_validators(
     })
 }
 
+/// Returns validators signing infos from remote chain
+/// * ***registered_query_id*** is an identifier of the corresponding registered interchain query
+pub fn query_validators_signing_infos(
+    deps: Deps<NeutronQuery>,
+    _env: Env,
```

Comment:
Let's remove unused variable from arguments?

Reply:
not sure it's possible, because other functions in this module contains this unused parameter as well and I am afraid that removal of it will break backward compatibility

Reply:
Ah, i see, that's not good.
Yeah, let's keep it, but will mark it as deprecated and remove in some of next releases